styles/style.css

```diff
@@ -1,192 +1,313 @@
 /* ========== Importing goole fonts ================== */
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital@1&display=swap');
 
-
-*{
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
 /* =============Navbar styling section====================== */
-.nav-bar ul{
-    display: flex;
-    background: #3C3A39;
-    /* border: 5px solid rgb(8, 8, 1); */
-    list-style-type: none;
-    width: 100vw;
-    height: 72px;
-    align-items: center;
-
-
+.nav-bar ul {
+  display: flex;
+  background: #3c3a39;
+  list-style-type: none;
+  width: 100vw;
+  height: 72px;
+  align-items: center;
 }
 
 /* ================== Styleling the flex elements ============================== */
-.nav-bar ul li{
-    flex: 1 1 100%;
-    text-decoration: none;
-    flex: 1;
-    
+.nav-bar ul li {
+  text-decoration: none;
+  flex: 1;
 }
 
 /* ================== hiding the hambuger icon in large screen ==================== */
-ul .hamburger-icon{
-    display: none;
+ul .hamburger-icon {
+  display: none;
 }
-.nav-bar ul li a{
-    text-decoration: none;
-    color: white;
-    font-family: Roboto;
-font-size: 18px;
-font-style: normal;
-font-weight: 700;
-line-height: 24px;
-letter-spacing: 0em;
-text-align: center;
 
+.nav-bar ul li a {
+  text-decoration: none;
+  color: white;
+  font-family: Roboto, Tahoma, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
 }
-ul #name{
-    flex: 5;
-    margin-left: 30px;
+
+ul #name {
+  flex: 5;
+  margin-left: 30px;
 }
 
 /* =========================styling the main container ===================== */
-.main-container{
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    background-color: #1C1A19;
-    padding-top: 5px;
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  padding-top: 5px;
+  max-height: 100vh;
 }
 
 /* =================== Styling for Max-screen of 650px ==================================== */
 @media screen and (max-width: 700px) {
```

Comment:
Please do not use media queries here as it's a mobile-first design. You will be required to use them later in the project.